fv3core/stencils/nh_p_grad.py

```diff
@@ -1,26 +1,23 @@
-from gt4py.gtscript import PARALLEL, computation, interval
+from gt4py.gtscript import PARALLEL, computation, interval, stencil
 
 import fv3core._config as spec
 import fv3core.stencils.a2b_ord4 as a2b_ord4
+import fv3core.utils.global_config as global_config
 import fv3core.utils.gt4py_utils as utils
-from fv3core.decorators import gtstencil
 from fv3core.utils.typing import FloatField, FloatFieldIJ
 
 
-@gtstencil()
 def set_k0(pp: FloatField, pk3: FloatField, top_value: float):
```

Comment:
if we are already in the process, can we make the name of this consistent with the other stencils?

Reply:
I think we are only using campy camel for class names, I don't think there's many if any other places in the code we use it for stencil definitions -- we could though! Maybe I missed the memo though?